front/admin/cli.ts

```diff
@@ -43,6 +44,11 @@ const workspace = async (command: string, args: parseArgs.ParsedArgs) => {
         sId: generateLegacyModelSId(),
         name: args.name,
       });
+      await GroupResource.makeNew({
```

Comment:
We should also create the workspace group no?

Reply:
Ok added, only issue is now my dumb fill script will throw ugly errors if run multiple times but I'm okay with it if you're okay with it

Reply:
I'm ok with it